vector/src/main/java/im/vector/app/features/home/room/list/home/HomeRoomListViewModel.kt

```diff
@@ -250,6 +270,38 @@ class HomeRoomListViewModel @AssistedInject constructor(
         }
     }
 
+    private fun getEmptyStateData(filter: HomeRoomFilter, selectedSpace: RoomSummary?): StateView.State.Empty? {
+        return when (filter) {
+            HomeRoomFilter.ALL ->
+                if (selectedSpace != null) {
+                    StateView.State.Empty(
+                            title = stringProvider.getString(R.string.home_empty_space_no_rooms_title, selectedSpace.displayName),
+                            message = stringProvider.getString(R.string.home_empty_space_no_rooms_message),
+                            image = drawableProvider.getDrawable(R.drawable.ill_empty_space),
+                            isBigImage = true
+                    )
+                } else {
+                    val userName = session.userService().getUser(session.myUserId)?.displayName ?: ""
+                    StateView.State.Empty(
+                            title = stringProvider.getString(R.string.home_empty_no_rooms_title, userName),
+                            message = stringProvider.getString(R.string.home_empty_no_rooms_message),
+                            image = drawableProvider.getDrawable(R.drawable.ill_empty_all_chats),
+                            isBigImage = true
+                    )
+                }
+            HomeRoomFilter.UNREADS ->
+                StateView.State.Empty(
+                        title = stringProvider.getString(R.string.home_empty_no_unreads_title),
+                        message = stringProvider.getString(R.string.home_empty_no_unreads_message),
+                        image = drawableProvider.getDrawable(R.drawable.ill_empty_unreads),
+                        isBigImage = true,
+                        imageScaleType = ImageView.ScaleType.CENTER_INSIDE
+                )
+            else ->
```

Comment:
We do not have empty state for `FAVOURITES` and `PEOPlE`? Maybe this is not possible to have those view if they are empty?
By the way there is a typo in `HomeRoomFilter.PEOPlE`. It should be `HomeRoomFilter.PEOPLE` (capital L)

Reply:
Thanks for confirming!